client/view/App.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import CreateElement from './createElement';
+import PureComponent from './pureComponent.jsx';
+import FunctionalComponent from './functionalCompoonent.jsx';
+
+export default class App extends React.Component {
+    render(){
+       return(
+           <div>
```

Comment:
Just as a practice let's use here <React.Fragment> (or <>). It's a good replacement of useless divs :)

Reply:
done

Reply:
:+1: